TGC/events/Filibuster.txt

```diff
@@ -0,0 +1,623 @@
+		country_event = {
+			id = 8870788
+			title = "EVTNAME8870788"
+			desc = "EVTDESC8870788"
+			picture = "wanderer_above_sea_of_fog"
+
+			trigger = {
+				tag = FIL
+				has_country_flag = filibuster_wars_active
+				war = no
+			}
+			
+			mean_time_to_happen = {
+				days = 1
+			}
+			fire_only_once = yes
+
+			option = {
+				name = "EVTOPTA8870788"
+				clr_country_flag = filibuster_wars_active
+				USA = { country_event=8870789 }
+				ai_chance = { factor = 0.85 }
+			}
+			option = {
+				name = "EVTOPTB8870788"
+				clr_country_flag = filibuster_wars_active
+				USA = { country_event=8870789 }
+				ai_chance = { factor = 0.1 }
+			}
+			option = {
+				name = "EVTOPTC8870788"
+				clr_country_flag = filibuster_wars_active
+				USA = { country_event=8870789 }
+				ai_chance = { factor = 0.05 }
+			}
+		}
+
+	## The Filibuster state asks to be annexed by the US
+		country_event = {
+			id = 8870789
+			title = "EVTNAME8870789"
+			desc = "EVTDESC8870789"
+			picture = "Cubanrebels"
+
+			is_triggered_only = yes
+
+			
+			mean_time_to_happen = {
+				days = 1
+			}
+			fire_only_once = yes
+
+			option = {
+				name = "EVTOPTA8870789"
+				FIL = { annex_to = USA }
+				badboy = 10
+
+				any_state = {
+					limit = {
+						is_slave = no
+						is_colonial = no
+						any_owned_province = { any_pop = { has_pop_culture = yankee } }
+					}
+					any_pop = {
+						limit = { NOT = { type = slaves } }
+						consciousness = 4
+						militancy = 2
+					}
+				}
+
+				any_state = {
+					limit = {
+						is_slave = yes
+						is_colonial = no
+						any_owned_province = { any_pop = { has_pop_culture = dixie } }
+					}
+					any_pop = {
+						limit = { NOT = { type = slaves } }
+						consciousness = -2
+						militancy = -1
+					}
+				}
+
+				set_global_flag = fil_annexed_by_usa
+
+				ai_chance = { factor = 0.05 }
+			}
+			option = {
+				name = "EVTOPTB8870789"
+				create_vassal = FIL
+				badboy = 5
+
+				any_state = {
+					limit = {
+						is_slave = yes
+						is_colonial = no
+						any_owned_province = { any_pop = { has_pop_culture = dixie } }
+					}
+					any_pop = {
+						limit = { NOT = { type = slaves } }
+						consciousness = -1
+					}
+				}
+
+				any_state = {
+					limit = {
+						is_slave = no
+						is_colonial = no
+						any_owned_province = { any_pop = { has_pop_culture = yankee } }
+					}
+					any_pop = {
+						limit = { NOT = { type = slaves } }
+						consciousness = 2
+						militancy = 1
+					}
+				}
+				ai_chance = { factor = 0.8 }
+			}
+			option = {
+				name = "EVTOPTC8870789"
+				prestige = -5
+				badboy = -2
+				ai_chance = { factor = 0.15 }
+			}
+		}
+
+	## The Filibuster state asks to be puppeted by the US
+		country_event = {
+			id = 8870790
+			title = "EVTNAME8870790"
+			desc = "EVTDESC8870790"
+			picture = "Cubanrebels"
+
+			is_triggered_only = yes
+			
+			mean_time_to_happen = {
```

Comment:
I'm pretty sure mean_time_to_happen does nothing if the event is set to triggered only.

Reply:
I might be wrong here though, do you know something I don't?